express-post-json/server.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+const app = express();
+
+const PORT = 8080;
+const GRADES_URL = '/api/grades';
+
+let nextID = 1;
+const grades = [];
+
+app.use(express.json());
+
+app.get(GRADES_URL, (req, res) => {
+  res.json(grades);
+});
```

Comment:
@jadenbertino Please store grades in an object with the id as their keys:

---

![image](https://github.com/jadenbertino/m2-0323-code-solutions/assets/16143771/8d2e0935-a19b-4c59-b686-d1d19cc8c165)